docs/reference/portal.dataProduct.json

```diff
@@ -595,6 +595,54 @@
                                     }
                                 }
                             ]
+                        },
+                        {
+                            "name": "monitoringSettings",
+                            "label": "Monitoring Settings",
+                            "type": "Microsoft.Common.Section",
+                            "visible": true,
+                            "elements": [
+                                {
+                                    "name": "monitoringSettingsText",
+                                    "type": "Microsoft.Common.TextBlock",
+                                    "visible": true,
+                                    "options": {
+                                        "text": "Specify Monitoring settings for this deployment.",
+                                        "link": {
+                                            "label": "",
+                                            "uri": ""
+                                        }
+                                    }
+                                },
+                                {
+                                    "name": "enableMonitoring",
+                                    "label": "Deploy Monitoring Components",
+                                    "type": "Microsoft.Common.OptionsGroup",
+                                    "visible": true,
+                                    "toolTip": "Deploy monitoring components like Log Analytics, Azure dashboards, alerts and diagnostic settings.",
+                                    "defaultValue": "No",
+                                    "constraints": {
+                                        "allowedValues": [
+                                            {
+                                                "label": "Yes",
+                                                "value": "yes"
+                                            },
+                                            {
+                                                "label": "No",
+                                                "value": "no"
+                                            }
+                                        ]
+                                    }
+                                },
+                                {
+                                    "name": "dataProductTeamEmail",
+                                    "label": "Alert Receivers Email ID",
+                                    "type": "Microsoft.Common.TextBox",
+                                    "visible": "[equals(steps('generalSettings').monitoringSettings.enableMonitoring, 'yes')]",
+                                    "defaultValue": "",
+                                    "toolTip": "Specify Email ID to receive monitoring alerts."
+                                }
```

Comment:
Can we add some regex validation here to make sure that an email was entered and also specify whether this is required or not?